ap_ruby_sdk/lib/ap_ruby_sdk/transaction.rb

```diff
@@ -7,7 +7,6 @@ class Transaction < ApiResource
     attr_accessor :customer,
                   :customerId,
                   :payment,
-                  :token,
```

Comment:
Because you never do transaction with token only, you do it with phone verification which wraps token and pin. Check out [documentation](http://www.alternativepayments.com/support/api/#create-transaction-using-token-and-pin).

So the flow is as:

1. Send PIN by SMS as explained [here](http://www.alternativepayments.com/support/api/#send-pin-by-sms)
2. Receive PIN over SMS on provided phone
3. [Create transaction](http://www.alternativepayments.com/support/api/#create-transaction-using-token-and-pin) with PIN from SMS and token received from API response in step 1

@AlternativePayments Am I right here?